upstairs/src/downstairs.rs

```diff
@@ -6053,19 +6051,19 @@ pub(crate) mod test {
         assert!(ds.in_progress(next_id, ClientId::new(2)).is_some());
 
         // Process resplies from the two running downstairs
-        let response = Ok(vec![]);
+        let response = || Ok(Default::default());
         ds.process_ds_completion(
             next_id,
             ClientId::new(0),
-            response.clone(),
+            response(),
```

Comment:
It's not better or worse in general, but I removed `derive(Clone)` from `RawReadResponse` so that we can't accidentally do an expensive buffer copy, so this was an easy way to generate multiple identical values without `clone()`.